modules/administration/src/lib/administration/components/admin-substep/admin-substep.component.ts

```diff
@@ -0,0 +1,93 @@
+import { AsyncPipe, CommonModule } from '@angular/common';
+import { Component, OnDestroy } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import { Store } from '@ngxs/store';
+import { ClientPaginatedTableComponent, ColumnCustom, Substep } from '@te44-front/shared';
+import { ConfirmationService } from 'primeng/api';
+import { ButtonModule } from 'primeng/button';
+import { ConfirmDialogModule } from 'primeng/confirmdialog';
+import { DialogService, DynamicDialogRef } from 'primeng/dynamicdialog';
+import { WorkflowState } from '../../../../state/workflow.state';
+import { AdminSubstepModalComponent } from '../admin-substep-modal/admin-substep-modal.component';
+import { WorkflowStateActions } from '../../../../state/actions/workflow.actions';
+
+@Component({
+  selector: 'app-admin-substep',
+  standalone: true,
+  imports: [RouterModule, CommonModule, ButtonModule, AsyncPipe, ConfirmDialogModule, ClientPaginatedTableComponent],
+  templateUrl: './admin-substep.component.html',
+  styleUrl: './admin-substep.component.less',
+  providers: [DialogService, ConfirmationService]
+})
+export class AdminSubstepComponent implements OnDestroy {
+  step$ = this.store.select(WorkflowState.getStep);
```

Comment:
je te conseil de faire un combineLatest pour te simplifier la gestion des pipe async 
`viewModel$ = combineLatest({
    step: this.store.select(WorkflowState.getStep),
    workflow: this.store.select(WorkflowState.getWorkflow)
  });`

et dans le html `*ngIf="viewModel$ | async as viewModel"`